core/message-tags-3.3.md

```diff
@@ -135,9 +135,14 @@ Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client
 ## Security considerations
 
 Client-only tags should be treated as untrusted data. They can contain any value
-and are not validated by servers in any way.
+and are not validated by servers in any way. The server MAY normalise values by applying the unescape
+and escape process.
```

Comment:
In this case I think it should explicitly state "unescape tag values when receiving data and escape those values before sending them out". 'normalise' isn't clear and raises the above questions, stating the exact process exactly when it can be applied should remove any ambiguity.